src/assets/scss/main.scss

```diff
@@ -93,3 +93,7 @@
     }
   }
 }
+
+.Family-Details {
```

Comment:
Why are there two files main.scss and main.css? And this class style is repeated in both these files. Please keep one